src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php

```diff
@@ -1873,6 +1874,16 @@ public function testNotifierWithoutTransports()
         $this->assertFalse($container->hasDefinition('texter'));
     }
 
+    public function testIfNotifierTransportsAreKnownByFrameworkExtension()
+    {
+        $container = $this->createContainerFromFile('notifier');
+
+        foreach ((new Finder())->in(\dirname(__DIR__, 4).'/Component/Notifier/Bridge')->directories()->depth(0)->exclude(['Mercure', 'SmsBiuras', 'MessageBird']) as $bridgeDirectory) {
```

Comment:
As soon as the repository symfony/message-bird-notifier is available:

```suggestion
        foreach ((new Finder())->in(\dirname(__DIR__, 4).'/Component/Notifier/Bridge')->directories()->depth(0)->exclude('Mercure') as $bridgeDirectory) {
```